Food_Tracker_App/app/src/main/res/layout/recipe_display.xml

```diff
@@ -6,6 +6,176 @@
     android:layout_height="match_parent"
     tools:context=".ui.recipes.RecipeDisplay">
 
+    <androidx.fragment.app.FragmentContainerView
```

Comment:
Elena is working on this, try not to push her code or merge your two PRs together